cmake/Config.cmake.in

```diff
@@ -0,0 +1,7 @@
+@PACKAGE_INIT@
+
+include(CMakeFindDependencyMacro)
+find_dependency(OpenAssetIO)
```

Comment:
Nice.

Reply:
For posterity, this was removed since it caused problems when OpenAssetIO was included as a subproject.